src/main/resources/templates/email/NewInterview_Interviewee.txt

```diff
@@ -0,0 +1,16 @@
+You have been registered for an interview on {{formatted_date}}. You are the interviewee. You will be be mock interviewed by {{interviewer_full_name}} for the position of: {{position}}.
```

Comment:
Security concern, Jack just wanted to have the interviewer's first name, not the full name.